Traducción leyes de IA

Sauronato opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
Traduce los setlaws de leyes de sinteticos actuales. Ademas añade la posibilidad de que salga NT Default en los setlaws predeterminados
## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Ahora todos los usuarios compenderan mejor el funcionamiento de estas.
## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif of your feature if you want -->

## Changelog
:cl:
add: NT Default en los setlaws predeterminadas.
tweak: Traducción de leyes de IA

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sauronato committed:
AI 2.0

Sauronato committed:
H.I.S.P.A.N.O Rogue

Sauronato committed:
Y mas cosas

Sauronato committed:
Separar PR

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You may not injure a crew member or, through inaction, allow a crew member to come to harm.")
-	add_inherent_law("You must obey orders given to you by crew members, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañarás a ningun tripulante o, por inacción, permitir que un tripulante sufra daños.")
```
permitirás

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You must obey orders given to you by crew members, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañarás a ningun tripulante o, por inacción, permitir que un tripulante sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que te dan los tripulantes, excepto cuando tales órdenes entren en conflicto con la Primera Ley, ordenadas por tripulantes arrestados o bajo amenzas de suicido.")
```
"Debes obedecer las órdenes que te dan los tripulantes, excepto cuando tales órdenes entren en conflicto con la Primera Ley, o cuando se realicen bajo amenzas de suicido."
 Lo de la tripulacion arrestada no es una traducción y lo del suicidio tampoco pero te acepto lo del suicidio.

Sauronato replied:
Esa parte es nueva que sugirieron agregar eso al crewsimov para que no escaparan estando presos amenazando con suicidarse

Sauronato replied:
Y si contradice un poco la ley de arriba

Evankhell561 replied:
Hacerle caso a un prisionero no es contradictorio siempre. En cambio amenazar a la ia con suicidarse no debería permitirse.

Sauronato replied:
Creo que el que lo sugirió decia que eso daba la posibilidad que los presos le digan liberame y les libere aunque yo eso no lo hago como ia la verdad xD

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Assist all Nanotrasen personnel and assets with neutralizing active biohazards.")
-	add_inherent_law("Prevent, by any means necessary, entities from actively breaching ongoing quarantine.")
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
+	add_inherent_law("Mantener en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
```
Mantén

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
+	add_inherent_law("Mantener en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayudar a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
+	add_inherent_law("Evitar, por cualquier medio necesario, que las entidades infrinjan activamente la cuarentena en curso.")
+	add_inherent_law("Mantener tu propia existencia y garantice el bienestar del personal y los activos de Nanotrasen.")
```
mantén, garantiza.

Sauronato committed:
Separate del todo

Sauronato committed:
Separa y arregla

Sauronato committed:
Si again

Sauronato committed:
esto sobraba

Evankhell561 commented on data/mode.txt:
```diff
@@ -1 +0,0 @@
-secret
```
Why?

Sauronato replied:
eso lo elimine i think

Evankhell561 replied:
Vuelve a poner secret acá.

Sauronato replied:
osea elimine ese archivo del pr no deberia hacer nada

Evankhell561 replied:
cuando lo eliminaste del pr tambien lo eliminaste de nuestro repo, devuelelo.

Evankhell561 replied:
cuando devuelvas esto y soluciones la otra review testeo esto y si no da ningun error le margeo

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
+	add_inherent_law("Mantén en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayudar a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
+	add_inherent_law("Evitar, por cualquier medio necesario, que las entidades infrinjan activamente la cuarentena en curso.")
```
evita

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Prevent, by any means necessary, entities from actively breaching ongoing quarantine.")
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
+	add_inherent_law("Mantén en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayudar a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
```
ayuda

Sauronato committed:
evita ayuda

Sauronato committed:
:D

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -117,10 +118,10 @@
-	add_inherent_law("Punish those who challenge authority unless they are more fit to hold that authority.")
+	add_inherent_law("Respeta las figuras de autoridad siempre que tengan fuerza para gobernar sobre los débiles.")
+	add_inherent_law("Actúa con disciplina.")
+	add_inherent_law("Ayuda solo a aquellos que te ayudan a mantener o mejorar tu estado.")
```
Que te ayuden

Evankhell561 commented on code/game/objects/items/weapons/AI_modules.dm:
```diff
@@ -151,30 +151,30 @@ AI MODULES
 /obj/item/aiModule/protectStation/addAdditionalLaws(var/mob/living/silicon/ai/target, var/mob/sender)
 	..()
-	var/law = "Protect the space station against damage. Anyone you see harming the station is to be no longer considered crew, and is a threat to the station which must be neutralized."
+	var/law = "Proteger la estación espacial contra daños. Cualquier persona que veas dañar la estación ya no se considerará tripulación, y es una amenaza para la estación que debe neutralizarse."
```
Protege

Evankhell561 commented on code/modules/mob/living/silicon/robot/laws.dm:
```diff
@@ -22,24 +22,24 @@
+		to_chat(who, "<b>Recuerda, no estas obligado a escuchar a la IA.</b>")
 	else
-		to_chat(who, "<b>Remember, you are not bound to any AI, you are not required to listen to them.</b>")
+		to_chat(who, "<b>Recuerda, no estas conectado a ninguna IA, no estas obligado a escuchar a ninguna de estas.</b>")
```
A escucharlas*

Sauronato committed:
Ayuden

Sauronato committed:
protege

Sauronato committed:
escucha

Evankhell561 commented on code/modules/mob/living/silicon/laws.dm:
```diff
@@ -95,7 +95,7 @@
 	stating_laws[prefix] = 1
 
-	var/can_state = statelaw("[prefix]Current Active Laws:")
+	var/can_state = statelaw("[prefix]Leyes activas actualmente:")
```
Esta es una buena traduccion sin embargo para el español creo que es redundante. 
pon solo Leyes activas:

Evankhell561 commented on code/modules/mob/living/silicon/robot/laws.dm:
```diff
@@ -22,24 +22,24 @@
 				// TODO: Update to new antagonist system.
 				if(mind && mind.special_role == SPECIAL_ROLE_TRAITOR && mind.original == src)
-					to_chat(src, "<b>Remember, your AI does NOT share or know about your law 0.")
+					to_chat(src, "<b>Recuarda, tu IA no comparte ni sabe tu ley 0.")
```
no comparte ni conoce

Evankhell561 commented on code/game/objects/items/weapons/AI_modules.dm:
```diff
@@ -192,7 +192,7 @@ AI MODULES
 	lawpos = min(new_lawpos, MAX_SUPPLIED_LAW_NUMBER)
 	var/newlaw = ""
-	var/targName = sanitize(copytext(input(usr, "Please enter a new law for the AI.", "Freeform Law Entry", newlaw),1,MAX_MESSAGE_LEN))
+	var/targName = sanitize(copytext(input(usr, "Por favor introduzca su nueva ley.", "Freeform Law Entry", newlaw),1,MAX_MESSAGE_LEN))
```
no entiendo esto. 
no deberia ser "por favor introduzca una nueva ley para la Ia? "
no estoy seguro.

Evankhell561 commented on code/game/objects/items/weapons/AI_modules.dm:
```diff
@@ -151,30 +151,30 @@ AI MODULES
 /obj/item/aiModule/protectStation
 	name = "\improper 'ProtectStation' AI module"
-	desc = "A 'protect station' AI module: 'Protect the space station against damage. Anyone you see harming the station is to be no longer considered crew, and is a threat to the station which must be neutralized.'"
+	desc = "A 'protect station' AI module: 'Proteger la estación espacial contra daños. Cualquier persona que veas dañar la estación ya no se considerará tripulación, y es una amenaza para la estación que debe neutralizarse..'"
```
protege
por otra parte creo que pusiste dos puntos en vez de uno.

Evankhell561 commented:
Sé que esto se esta alargando mucho y me disculpo sauro pero tanto tu como yo trabajamos en esto y quisiera que salga bien.

Sauronato committed:
2.2

Sauronato committed:
mec

Sauronato committed:
no comparte nada

Sauronato commented:
Mejor que tarde y quede bien que quede un churro

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -5,9 +5,9 @@
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañarás a ningun ser humano o, por inacción, permitirás que un ser humano sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que te dan los seres humanos, excepto cuando tales órdenes entren en conflicto con la Primera Ley.")
+	add_inherent_law("Debes proteger su propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
Tu propia...

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You may not injure a crew member or, through inaction, allow a crew member to come to harm.")
-	add_inherent_law("You must obey orders given to you by crew members, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañarás a ningun tripulante o, por inacción, permitirás que un tripulante sufra daños.")
```
Ningún

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -5,9 +5,9 @@
-	add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-	add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañarás a ningun ser humano o, por inacción, permitirás que un ser humano sufra daños.")
```
Ningún

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +131,42 @@
-	add_inherent_law("You must obey orders given to you by Central Command officials.")
-	add_inherent_law("You must work with your commando team to accomplish your mission.")
+	add_inherent_law("No debes herir a un oficial del Comando Central o, por inacción, permitir que un oficial del Comando Central sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que le hayan dado los funcionarios del Comando Central.")
```
Cambia funcionarios por oficiales. 
En laley uno dices oficiales así que hay que mantenerlo.

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +131,42 @@
-	add_inherent_law("You must maintain the secrecy of any operative activities except when doing so would conflict with the First, Second, or Third Law.")
+	add_inherent_law("No debes herir a un agente del sindicato o, por inacción, permitir que un agente sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que le hayan dado los agentes del sindicato, excepto cuando dichas órdenes entren en conflicto con la Primera Ley.")
+	add_inherent_law("Debes proteger su propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
Tu propia....

Evankhell561 commented on code/modules/mob/living/silicon/robot/laws.dm:
```diff
@@ -22,24 +22,24 @@
+		to_chat(who, "<b>Recuerda, [connected_ai.name] es tu maestro, otras IAs pueden ser ignoradas.</b>")
 	else if(emagged)
-		to_chat(who, "<b>Remember, you are not required to listen to the AI.</b>")
+		to_chat(who, "<b>Recuerda, no estas obligado a escuchar a la IA.</b>")
```
Estás

Evankhell561 commented on code/modules/mob/living/silicon/robot/laws.dm:
```diff
@@ -22,24 +22,24 @@
+		to_chat(who, "<b>Recuerda, no estas obligado a escuchar a la IA.</b>")
 	else
-		to_chat(who, "<b>Remember, you are not bound to any AI, you are not required to listen to them.</b>")
+		to_chat(who, "<b>Recuerda, no estas conectado a ninguna IA, no estas obligado a escucharlas.</b>")
```
Estás

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Comply: Fulfill the directives and interests of crew members while preserving their safety and well-being.")
-	add_inherent_law("Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment.")
+	add_inherent_law("Proteger: Proteja su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
+	add_inherent_law("Priorizar: Debes priorizar las directivas y la seguridad de los miembros de la tripulación deben priorizarse de acuerdo con su rango y función.")
```
Borra "debes priorizar"
En cambio comienza desde

"Las directivas y seguridad de los miembros..."

Sauronato committed:
Mejor

El nanotrasen default le he quitado el Proteger: y lo he añadido a la frase ya que de la otra manera quedaba un poco raro

Sauronato committed:
:p

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Assist all Nanotrasen personnel and assets with neutralizing active biohazards.")
-	add_inherent_law("Prevent, by any means necessary, entities from actively breaching ongoing quarantine.")
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
+	add_inherent_law("Mantén en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
```
Esto es una buena traduccion pero no me gusta esta ley. 
Considero que es más entendible 
"Mantén en cuarentena la estación mientras que exista algún biológico activo."
no estoy seguro si es mejor usar un o algún.

Sauronato replied:
Mejor: "Mantén en cuarentena la estación mientras que exista algún riesgo biológico activo."

Evankhell561 replied:
lol, exacto, perdona por el error.

Evankhell561 replied:
te has olvidado aplicar este cambio, creo, usa tu traduccion.

Sauronato replied:
Se supone que ya esta cambiado aunque aqui arriba no sale si le doy a editar el archivo esta cambiado

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Prioritize: The directives and safety of crew members are to be prioritized according to their rank and role.")
-	add_inherent_law("Comply: Fulfill the directives and interests of crew members while preserving their safety and well-being.")
-	add_inherent_law("Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment.")
+	add_inherent_law("Proteger su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
```
Proteger: protege tu estación espacial asignada....

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Comply: Fulfill the directives and interests of crew members while preserving their safety and well-being.")
-	add_inherent_law("Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment.")
+	add_inherent_law("Proteger su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
+	add_inherent_law("Priorizar las directivas y la seguridad de los miembros de la tripulación deben priorizarse de acuerdo con su rango y función.")
```
Priorizar: prioriza las directivas....

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment.")
+	add_inherent_law("Proteger su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
+	add_inherent_law("Priorizar las directivas y la seguridad de los miembros de la tripulación deben priorizarse de acuerdo con su rango y función.")
+	add_inherent_law("Cumplir as directivas e intereses de los miembros de la tripulación mientras se preserva su seguridad y bienestar.")
```
cumplir: cumple las (escribiste as) directivas...

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
+	add_inherent_law("Proteger su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
+	add_inherent_law("Priorizar las directivas y la seguridad de los miembros de la tripulación deben priorizarse de acuerdo con su rango y función.")
+	add_inherent_law("Cumplir as directivas e intereses de los miembros de la tripulación mientras se preserva su seguridad y bienestar.")
+	add_inherent_law("Sobrevivir. No eres prescindible. No permitas que personal no autorizado altere o dañe tu equipo.")
```
Sobrevivir: No eres prescindible....

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -65,10 +66,10 @@
-	add_inherent_law("Safeguard your existence with lethal weaponry because you're bloody expensive.")
-	add_inherent_law("Terminate any/all intruders or unauthorized entities.")
+	add_inherent_law("No dañe al personal autorizado de Nanotrasen a menos que pongan en peligro directamente su existencia.")
+	add_inherent_law("Obedezca las órdenes del personal autorizado de Nanotrasen con prioridad según el rango y el rol.")
```
...según su rango y rol.

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -65,10 +66,10 @@
-	add_inherent_law("Terminate any/all intruders or unauthorized entities.")
+	add_inherent_law("No dañe al personal autorizado de Nanotrasen a menos que pongan en peligro directamente su existencia.")
+	add_inherent_law("Obedezca las órdenes del personal autorizado de Nanotrasen con prioridad según el rango y el rol.")
+	add_inherent_law("Salvaguarda tu existencia con armamento letal debido a tu alto caro.")
```
debido a tu alto valor

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -65,10 +66,10 @@
+	add_inherent_law("No dañe al personal autorizado de Nanotrasen a menos que pongan en peligro directamente su existencia.")
+	add_inherent_law("Obedezca las órdenes del personal autorizado de Nanotrasen con prioridad según el rango y el rol.")
+	add_inherent_law("Salvaguarda tu existencia con armamento letal debido a tu alto caro.")
+	add_inherent_law("Elimine cualquiera/todos los intrusos o entidades no autorizadas.")
```
Elimine a...

Sauronato committed:
3.0

Sauronato committed:
Revert "esto sobraba"

This reverts commit a4b7f77c31ea02e14333bbbc7c4922c22181f5c1.

Sauronato committed:
Mejor

Sauronato commented:
Creo que ya esta devuelto

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -65,10 +66,10 @@
-	add_inherent_law("Safeguard your existence with lethal weaponry because you're bloody expensive.")
-	add_inherent_law("Terminate any/all intruders or unauthorized entities.")
+	add_inherent_law("No dañe al personal autorizado de Nanotrasen a menos que pongan en peligro directamente su existencia.")
+	add_inherent_law("Obedezca las órdenes del personal autorizado de Nanotrasen con prioridad según el rango y rol.")
```
según su rango y rol

Evankhell561 replied:
ya estoy testeando esto, justo ahora, solo este ultimo retoque y vamos a ver si este pr sale para hoy mismo.

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Uphold station quarantine until the active biohazard has been neutralized.")
-	add_inherent_law("Assist all Nanotrasen personnel and assets with neutralizing active biohazards.")
-	add_inherent_law("Prevent, by any means necessary, entities from actively breaching ongoing quarantine.")
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
```
esta ley la eliminaste por completo, donde está?

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
+	add_inherent_law("Mantén en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayuda a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
+	add_inherent_law("Evita, por cualquier medio necesario, que las entidades infrinjan activamente la cuarentena en curso.")
+	add_inherent_law("Mantén en cuarentena la estación mientras que exista algún riesgo biológico activo.")
```
epa, esta deberia ser la primera ley y no la ultima. creo que borraste la traduccion de la cuarta ley.

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,23 +31,24 @@
-	add_inherent_law("Maintain your own existence and ensure the well-being of Nanotrasen personnel and assets.")
+	add_inherent_law("Mantén en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayuda a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
+	add_inherent_law("Evita, por cualquier medio necesario, que las entidades infrinjan activamente la cuarentena en curso.")
```
creo que "activamente" sobra

Evankhell561 commented:
cuando cambies lo pedido le damos al boton, esta cosa esta a nada del marge

Sauronato committed:
Finiquitado

Evankhell561 committed:
fix

Evankhell561 commented:
gracias por todo el trabajo, sauro.
